src/treetensornetworks/projttns/abstractprojttn.jl

```diff
@@ -65,27 +65,15 @@ function _separate_first_two(V::Vector)
   return frst, scnd, rst
 end
 
+projected_operator_tensors(P::AbstractProjTTN) = error("Not implemented.")
+
+#contract(P::AbstractProjTTN, v::ITensor)::ITensor = foldl(*,projected_operator_tensors(P);init=v)
 function contract(P::AbstractProjTTN, v::ITensor)::ITensor
```

Comment:
```suggestion
function contract(P::AbstractProjTTN, v::ITensor)
```
and same thing elsewhere.